lib/content-services/search/components/search-sorting-picker/search-sorting-picker.component.spec.ts

```diff
@@ -0,0 +1,104 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SearchSortingPickerComponent } from './search-sorting-picker.component';
+import { SearchQueryBuilderService } from '../../search-query-builder.service';
+import { AppConfigService } from '@alfresco/adf-core';
+
+describe('SearchSortingPickerComponent', () => {
```

Comment:
Those tests are actually test the methods can we test the behaviours ? And do we need that this component is so bounded with the search instead to have a one general for the datatable?